app/components/windows/SourcesShowcase.vue

```diff
@@ -262,6 +251,11 @@ h2 {
   background-color: @day-section;
   width: 49%;
   .radius();
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  max-width: 100%;
+  display: inline-block;
+  overflow: hidden;
```

Comment:
i thought width needed a pixel measurement here to work with ellipsis, is it ok like this?